new-frontiers-14/frontier-station-14
# Stasis Directional Fan Update

## About the PR



This PR is waiting on #1965 , when the resources from that one are available I'll perform the final update here. I wanted to push this one a bit early while I was still waiting so it had some time to gather eyes.

- Adds directional fans to the docking and egress points.
- Updates the reactor type to uranium. Double plasma is bad.
- Swapped out the plasma fuel locker for uranium.
- Alteration to the superstructure to allow for a safety cubby for the super pacman. I know it's not required but it seems appropriate.
- Added a CMO stole to the clothing pile.
- Added a new chemistry shelf to the chem lab, because they're pretty neat.
- Added a new shelf of the main medical room, because they're pretty neat.
- Moved the medicinal seed crate to the botany lab.
- Botany lab slightly increased in size to match the reactor room. 
- Added one hydroponic tray to even out the space. If this seems excessive let me know.
- Rewired the main medical bay to allow for the main power hungry equipment to be controlled of their own APC.
- Slight adjustment to piping in the engineering bay to account for the new generator cubby. 
- Vent/scrubber added to docking airlock.
- Fluff changes to the guidebook communicating the new fictional shipyard company connected to the posters.


## Why / Balance


- Directional fans are the future.
- Double plasma is bad.
- Better player management over power is desirable. 
- Shelves are good.

## How to test


- Check all vents/scrubbers.
- Check cryo tubes for gas. Test freezer temp drop.
- Check all power systems for charge.
- Check power room radiation shutter.
- Check main hall window shutters.
- Open/Close the new medical shelf in the chem lab and main hall, attempt to place items.
- Test cryo area APC. Use camera tools to ensure all thrusters remain on while the cryo tubes and stasis beds deactivate.

## Media

Pipe dream.
![ss+(2024-09-10+at+06 07 34)](https://github.com/user-attachments/assets/d5283638-a951-4f3d-97b2-5bc268d5b602)
Normal Function
![ss+(2024-09-10+at+06 08 08)](https://github.com/user-attachments/assets/dff5d5ac-1bb8-4c24-9586-de7b102b58b7)
Cryo power off.
![ss+(2024-09-10+at+06 08 21)](https://github.com/user-attachments/assets/3f90f99e-0fe3-4b0a-9fc4-38aed3b31366)


## Requirements


## Breaking changes


Hopefully none here. However if this goes through more quickly than I expect it might have a weird reaction with #1918 

**Changelog**



🆑 
- add: Added directional fans to the Stasis.
- tweak: Changed Stasis power systems to uranium.
